panda/src/express/weakPointerTo.I

```diff
@@ -45,7 +45,6 @@ WeakPointerTo(const WeakPointerTo<T> &copy) :
 template<class T>
 INLINE TYPENAME WeakPointerTo<T>::To &WeakPointerTo<T>::
 operator *() const {
-  nassertr(!this->was_deleted(), *((To *)NULL));
```

Comment:
Resolution: `assert`

(And similar for the operators that return a NULL pointer, which is just going to be dereferenced immediately by the caller - e.g. `operator ->`)